### ISSUE TITLE
api.GPU - "EdgeOS" supported for Edge WebGPU

### ISSUE THREAD

**Issue by vgskye** [mdn/browser-compat-data#20002]
### What type of issue is this?

Incorrect support data (example: Chrome says "86" but support was added in "40")

### What information was incorrect, unhelpful, or incomplete?

In the edge implementation notes, it says that it's only supported on EdgeOS, macOS, and Windows.
EdgeOS is, from what I can gather, an OS for a certain lineup of routers.

### What browsers does this problem apply to, if applicable?

Chromium (Chrome, Edge 79+, Opera, Samsung Internet)

### What did you expect to see?

A router OS that cannot run Edge or even any browser is not listed as a supported platform.

### Did you test this? If so, how?

No, I do not own an Ubiquiti EdgeRouter, but it is not even listed as a supported platform in the downloads page for Edge.

### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/WebGPU_API

### MDN metadata

<details>
<summary>MDN page report details</summary>

* Query: `api.GPU`
* Report started: 2023-05-31T12:15:37.816Z

</details>

**Comment by Elchi3** [comment]
This should be fixed by https://github.com/mdn/browser-compat-data/pull/19902.

Not sure why it isn't deployed yet. Did we miss anything, @queengooborg?

**Comment by queengooborg** [comment]
I just did a bunch of testing, and this seems to be an issue on Yari's end.  I added a new unittest (https://github.com/mdn/browser-compat-data/pull/20020) to make sure our mirroring is working correctly, and it is.  I also just checked the latest BCD release, and its notes are working fine:

<img width="1137" alt="Screenshot 2023-06-03 at 18 06 35" src="https://github.com/mdn/browser-compat-data/assets/5179191/741f95f7-44ca-4529-9a8b-ddc95056fb90">

So, for whatever reason, Yari hasn't yet updated.

I'm going to close this issue because it's not an issue with BCD, but I'll report this to the engineers to hopefully get it fixed.
